JustSaying/AwsTools/MessageHandling/MessageResultLogger.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace JustSaying.AwsTools.MessageHandling
+{
+    /// <summary>
+    /// Handler allowing you to access the SQS or SNS message response.
+    /// </summary>
+    /// <param name="result">The result of the SNS notification or SQS enqueue.</param>
+    /// <returns></returns>
+    public delegate void MessageResultLogger(MessageResult result);
```

Comment:
Could you just use `Action<MessageResult>` instead of having to declare a custom delegate type?

Reply:
Yeah, this is going